<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>目录层级数值：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_DirNum"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelLoc">
         <property name="text">
          <string>选择位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenDir">
         <property name="text">
          <string>生成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>文件数量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_FileNum"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelFile">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenFile">
         <property name="text">
          <string>生成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>文件名长度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_FileNameLen"/>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtn_txt">
         <property name="text">
          <string>txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtn_jpeg">
         <property name="text">
          <string>jpeg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelLocForFileNameLen">
         <property name="text">
          <string>选择位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenFile_NameLen">
         <property name="text">
          <string>生成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>文件大小(MB)：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_FileFixSize"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelLocForFileSize">
         <property name="text">
          <string>选择位置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenFile_FileFixSize">
         <property name="text">
          <string>生成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
